src/Stratis.Bitcoin.Tests/BlockPulling/BlockPullerBehaviorTests.cs

```diff
@@ -21,8 +21,7 @@ public BlockPullerBehaviorTests()
             var ibdState = new Mock<IInitialBlockDownloadState>();
             ibdState.Setup(x => x.IsInitialBlockDownload()).Returns(() => true);
 
-            var loggerFactory = new ExtendedLoggerFactory();
-            loggerFactory.AddConsoleWithFilters();
+            var loggerFactory = ExtendedLoggerFactory.Create();
```

Comment:
I would actually love to get rid of this hack `ExtendedLoggerFactory` we just drop prefix stuff from logs it's not so crucial anyway